WireUI/Tests/WireMainNavigationUITests/Coordinator/AnyMainCoordinatorTests.swift

```diff
@@ -0,0 +1,184 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import XCTest
+
+@testable import WireMainNavigationUI
+
+final class AnyMainCoordinatorTests: XCTestCase {
+
+    private var mockMainCoordinator: MockMainCoordinatorProtocol!
+    private var sut: AnyMainCoordinator<MockMainCoordinatorDependencies>!
+
+    override func setUp() {
+        mockMainCoordinator = .init()
+        sut = .init(mainCoordinator: mockMainCoordinator)
+    }
+
+    override func tearDown() {
+        sut = nil
+        mockMainCoordinator = nil
+    }
+
+    @MainActor
```

Comment:
at that point, is it worth making AnyMainCoordinatorTests a mainActor?

Reply:
fixed: c4f678e363e2de2ff75c35d47bd9aa1da2ef8a02